### ISSUE TITLE
SA1606 inheritdoc analysis does not consider overrides

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1944]
The following code should report SA1606, but no warning is reported.

```csharp
/// <summary>
/// </summary>
/// <inheritdoc/>
public string Property => "P";
```
